src/main/java/dinkplugin/message/DiscordMessageHandler.java

```diff
@@ -145,4 +156,21 @@ public void onResponse(@NotNull Call call, @NotNull Response response) {
             }
         });
     }
+
+    private static void injectContent(@NotNull NotificationBody<?> body, boolean screenshot) {
+        List<Embed> embeds = new ArrayList<>(body.getEmbeds() != null ? body.getEmbeds() : Collections.emptyList());
+        embeds.add(0,
+            Embed.builder()
+                .author(new Author(body.getPlayerName()))
+                .color(Utils.PINK)
```

Comment:
Should we do a different color per notifier type instead?

Reply:
I don't know if I could connect a color to a notifier (i.e. loot is yellow, combat achievments are green) so we'd probably end up picking them at random which feels off so unless someone has a good scheme for this I'd say just keep it single-color for now.